### PR TITLE
fix(number-input-pinwheel): Remove pinwheel arrows that were showing …

### LINKED ISSUE #171: Remove "pinwheel" that shows in the amount field of the pay/request form
Get rid of the pinwheel arrows that show next to the amount field in the pay/request form.  Seen on Windows/Ubuntu for me.  This was reported by a couple other users as well.

pinwheel

### PR DESCRIPTION
**Pull request by mkehlmann**
…in the amount field of the request and pay forms

Fixes #171

### CODE REVIEW

File: app/components/Form/PayForm.scss [hunk 1]
```diff
@@ -172,4 +172,10 @@
       cursor: pointer;
     }
   }
+}
+
+input[type=number]{
```
Comment (JimmyMow) [inline app/components/Form/PayForm.scss]: sorry super small but can you add a space between `input[type=number]` and `{`?

Comment (mkehlmann) [reply app/components/Form/PayForm.scss]: Sure thing, I'll update it shortly.

**Comment by mkehlmann** [comment]
Looks like "ChromeDriver" had a little hiccup starting up for the Travis Build:

"ChromeDriver did not start within 5000ms"

Is there a way to kick off travis manually, or will you wait until there is something else from master to merge in to kick it off again?

**Comment by Empact** [comment]
@mkehlmann You can rebase the branch or modify on of the commit messages. Either will produce a new sha for the branch which CI will re-run against.

**Comment by JimmyMow** [comment]
Nice. I used Zap on Windows for the first time today and realized those pinbars are everywhere. I might make this change in the app.global.scss file, but this PR is good for now :)

**Pull request merged**